refocus-bdk-server.js

```diff
@@ -362,7 +374,7 @@ module.exports = (config) => {
     });
 
     socket.on('connect', () => {
-      socket.emit('auth', token);
+      socket.emit('auth', BOT_INSTALL_TOKEN);
```

Comment:
What's the difference between these two?

Reply:
token gets overridden, it's originally the bot install token but then becomes the socket authorisation token which will fail authentication.

Reply:
Oh, so BOT_INSTALL_TOKEN is a user token and token is a bot token, and the realtime app only accepts user tokens for auth.

If I'm reading this correctly, token is only used here, so once this has been rolled out and the toggle removed the bot token will no longer be used at all? (Or is the bot token also used in bot-specific code?)

So this change means it will no longer be using a bot-specific token for auth. Is that correct? Or do we need to support using bot tokens for the realtime app? What is the purpose of having a separate bot token in the first place?

@ConalCosgrove @iamigo

Reply:
Yes, as far as I am aware once we switch over to the realtime app we won't need the bot-specific token. 

@BrandonMurnane could you confirm this? Is the socket token needed for anything other than authentication (which will use the API token in the realtime app)?

Reply:
socket token is just for auth im p sure